samples/holiday-package-sample/README.md

```diff
@@ -0,0 +1,112 @@
+# Online Holiday Packages Booking System.
+
+This scenario is about an online Holiday Package Booking System.
+This is a sample which offers a quick reservation system which involves minimum efforts. This sample will show how
+WSO2 EI is capable of considering a mixture of numerous holiday properties (Flight/ Car/ Hotel) to provide the client with the most suitable holiday package.
+Client provides departure date, return date, from, to and the vehicle type he needs in the request. Service will send booking requests parallely to multiple Airline companies, vehicle rental companies and hotels.
+Service will include the flight from the first airline to respond and the vehicle from the company which is offering cheapest price to the holiday package.
```

Comment:
Shall modify this to get the cheapest airline and first car rental service to respond ?